timescaledb/tutorials/building-a-custom-timescaledb-dashboard.md

```diff
@@ -0,0 +1,326 @@
+chunk_name        | _hyper_2_2_chunk
+range_start       | 2021-04-29 00:00:00+00
+range_end         | 2021-05-06 00:00:00+00
+```
+
+The chunk name that is returned with the dataset is unique and will be used in other queries to retrieve enhanced details about each chunk for visualization purposes. In this example, you can see that the chunk has a `range_start` and `range_end` that spans one week. As new data is inserted into this table, any data that has a timestamp between 2021-04-29 and 2021-05-06  will be stored on this specific chunk for the `conditions` table.
+
+### Get details about compression status for a chunk
+
+Once we know the name and time range of each chunk, the second challenge is to get more detail about the compression status and how much disk we’re saving by compressing the data. We can get this additional information by querying the `chunk_compression_stats` function with the `conditions` hypertable:
+
+```sql
+tsdb=> SELECT * FROM chunks_detailed_size('conditions');
+-[ RECORD 1 ]------------------+----------------------
+chunk_schema                   | _timescaledb_internal
+chunk_name                     | _hyper_6_913_chunk
+compression_status             | Compressed
+before_compression_table_bytes | 204800
+before_compression_index_bytes | 360448
+before_compression_toast_bytes | 0
+before_compression_total_bytes | 565248
+after_compression_table_bytes  | 8192
+after_compression_index_bytes  | 16384
+after_compression_toast_bytes  | 98304
+after_compression_total_bytes  | 122880
+node_name                      |
+-[ RECORD 2 ]------------------+----------------------
+chunk_schema                   | _timescaledb_internal
+chunk_name                     | _hyper_6_880_chunk
+compression_status             | Uncompressed
+before_compression_table_bytes |
+before_compression_toast_bytes |
+before_compression_total_bytes |
+after_compression_table_bytes  |
+after_compression_index_bytes  |
+after_compression_toast_bytes  |
+after_compression_total_bytes  |
+node_name                      |
+```
+
+### Get compression stats and size
+
+One detail you might notice is that when the chunk is uncompressed, it does not bring the size of the chunk. For the uncompressed data, we need to fall back to the `chunks_detailed_size` function, passing a specific hypertable name as a parameter:
+
+```sql
+tsdb=> SELECT * FROM chunks_detailed_size('conditions');
+-[ RECORD 1 ]+----------------------
+chunk_schema | _timescaledb_internal
+chunk_name   | _hyper_6_853_chunk
+table_bytes  | 8192
+index_bytes  | 40960
+toast_bytes  | 98304
+total_bytes  | 147456
+node_name    |
+```
+
+From this function we can use the `total_bytes` when we know that the chunk is not compressed.
+
+### Building views for our TimescaleDB metrics
+
+Now that we know where to get all of the data we need to drive the visualization, it’s time to join it together in a view that can be queried using SQL (and eventually, our application).
```

Comment:
```suggestion
Now that you know how to gather all of the data you need to drive the visualization, it’s time to join it together in a view that can be queried using SQL (and eventually, our application).
```

Reply:
thanks a lot @Loquacity for all the suggestions from `we -> you` I'm still getting used to who is who here 😄